src/hotspot/share/runtime/globals.hpp

```diff
@@ -949,9 +949,6 @@ const int ObjectAlignmentInBytes = 8;
           "null (+offset) will not raise a SIGSEGV, i.e.,"                  \
           "ImplicitNullChecks don't work (PPC64).")                         \
                                                                             \
-  product(bool, EnableThreadSMRExtraValidityChecks, true, DIAGNOSTIC,       \
-             "Enable Thread SMR extra validity checks")                     \
```

Comment:
Do you still want to make this change as part of this PR?

Reply:
Yes I would. All the analysis is recorded in this bug and the PR.